### ISSUE TITLE
Installing Widget doesn't work with Yarn

### ISSUE THREAD

**Issue by mraible** [okta/okta-signin-widget#191]
```bash
yarn add @okta/okta-signin-widgetyarn add v0.21.3
[1/4] 🔍  Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@okta%2fci-update-package: Not found".
info If you think this is a bug, please open a bug report with the information provided in "/Users/mraible/angular-okta-example/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

It works fine with npm.

**Comment by joetastic** [comment]
Having the same issue only missing `https://registry.yarnpkg.com/@okta%2fci-pkginfo`

**Comment by mraible** [comment]
It doesn't seem like Yarn supports installing dependency without optional dependencies: https://github.com/yarnpkg/yarn/issues/2987

**Comment by rchild-okta** [comment]
Fixed with #249
